Ical.Net/DataTypes/RecurrencePattern.cs

```diff
@@ -93,11 +98,12 @@ public RecurrenceRestrictionType RestrictionType
                 {
                     return _restrictionType.Value;
                 }
-                return Calendar?.RecurrenceRestriction ?? RecurrenceRestrictionType.Default;
+                return RecurrenceRestrictionType.NoRestriction;
```

Comment:
That's a breaking change I assume, right?

Reply:
That should **not** have been left in the code. Thanks!
I will revert the change and set `RecurrenceRestrictionType.NoRestriction` in the tests that would fail otherwise.